src/components/Common/List.vue

```diff
@@ -14,14 +14,19 @@
       :documents="documents"
       :total-documents="totalDocuments"
       :display-bulk-delete="displayBulkDelete"
+      :display-create="displayCreate"
       :all-checked="allChecked"
       :selected-documents="selectedDocuments"
       :length-document="selectedDocuments.length">
 
         <div class="collection">
           <div class="collection-item" transition="collection" v-for="document in documents">
-            <component :is="itemName" @checkbox-click="toggleSelectDocuments" :document="document"
-                       :is-checked="isChecked(document.id)"></component>
+            <component :is="itemName"
+                       @checkbox-click="toggleSelectDocuments"
+                       :document="document"
+                       :is-checked="isChecked(document.id)"
+                       :rights="rightsOnDocument">
```

Comment:
I don't understand why you are passing this in props ?

Each child component must handle their own policies.
You can directly call the `userAuthorization` service in the child component.